suplemon/file.py

```diff
@@ -141,6 +141,7 @@ def _read_text(self, file):
             f.close()
             return data
         except:
+            self.logger.exception("Reading file failed.")
```

Comment:
Thanks, good catch! I'll change this in the next version. I accidentally released from the dev branch so I had to merge to get master up to date.